### PR TITLE
Multiplelogin

### PR DESCRIPTION
**Pull request by jonay2000**

### CODE REVIEW

File: cshub-server/src/db/entities/emaildomain.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    @Expose()
+    @Column({
+        type: "varchar",
+        length: 64,
+        unique: true
+    })
+    domain!: string;
+
+    @OneToMany(type => User, user => user.domain)
+    users?: User[];
```
Comment (NULLx76) [inline cshub-server/src/db/entities/emaildomain.ts]: Shouldn't a user have a domain instead of a domain having users?

Comment (jonay2000) [reply cshub-server/src/db/entities/emaildomain.ts]: good point

Comment (jonay2000) [reply cshub-server/src/db/entities/emaildomain.ts]: that's why it wasn't working

Comment (RobbinBaauw) [reply cshub-server/src/db/entities/emaildomain.ts]: I'll take a look at this later, but this should be fine. You should make these joins both ways in typeorm

**Pull request merged**